packages/donations-app/app/components/checkout/checkoutService.js

```diff
@@ -0,0 +1,167 @@
+angular.module('aliceApp')
+  .service('CheckoutService', ['$rootScope', '$uibModal', 'MANGO', 'AuthService', 'NotificationService', '$http', 'API', '$q', function ($rootScope, $uibModal, MANGO, AuthService, NotificationService, $http, API, $q) {
+
+    // Set MANGOPAY API base URL and Client ID
+    mangoPay.cardRegistration.baseURL = MANGO.url;
+    mangoPay.cardRegistration.clientId = MANGO.clientId;
+
+    const MANGO_TEST_CARD = "4706750000000009";
+    const MANGO_TEST_CARD_3DS = "3569990000000157"; // For testing of payments over 50EUR
```

Comment:
This constant (MANGO_TEST_CARD_3DS) is not used

Reply:
Yes, but I left it for local testing, let it stay pls so we don't need to check mangoPay docs.